zephyr/port/bootloader/bootloader_flash.c

```diff
@@ -30,6 +30,13 @@
 
 #include "esp_rom_spiflash.h"
 
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+
+#define DT_DRV_COMPAT espressif_esp32_flash_controller
+#define SOC_NV_FLASH_NODE DT_INST(0, soc_nv_flash)
+#define FLASH_BLK_SZ DT_PROP(SOC_NV_FLASH_NODE, write_block_size)
```

Comment:
This is usually small buffer.. up to 32 bytes. We could increase it to 256 byte-long as well.